en/releases/centreon-commercial-extensions.md

```diff
@@ -273,6 +273,13 @@ If you have feature requests or want to report a bug, please contact support.
 
 ## Centreon Auto Discovery
 
+### 20.04.8
+
```

Comment:
Missing this ticket [HD/Provider/Install/Upgrade] Limit provider definition version that is imported to 2.0 in 20.10/20.04